### PR TITLE
Drawing button redirecting to the old wkp.

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/Permalink/Permalink.js [hunk 1]
```diff
@@ -44,6 +45,33 @@ const defaultProps = {
...
+      'EPSG:3857',
+      'EPSG:21781',
+    );
+    urlParams.x = newX;
+    urlParams.y = newY;
+  }
+
+  urlParams['wkp.draw'] = drawId;
+
+  window.location.href = `http://wkp.prod.trafimage.geops.ch/?debug#/ch.sbb.netzkarte.draw?${qs.stringify(
```
Comment (mhaertwig) [inline src/components/Permalink/Permalink.js]: No `?debug` and please make this configurable.

Comment (edellucien) [reply src/components/Permalink/Permalink.js]: I use configurable 'appBaseUrl'  now

File: src/layers/layerHelper.js [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+// Allow to convert zoom from new wkp to old one.
+function convertToOldZoom(zoom) {
+  if (zoom < 9) {
+    return 1;
+  }
+  if (zoom > 18) {
+    return 10;
+  }
+  return zoomEquivalent[zoom];
+}
```
Comment (gevos) [inline src/layers/layerHelper.js]: @edellucien What about decimal numbers as zoom value?

File: src/components/Permalink/Permalink.js [hunk 3]
```diff
@@ -63,6 +94,11 @@ class Permalink extends PureComponent {
       ...initialState,
     };
 
+    if (parameters['wkp.draw']) {
+      // Redirection to old wkp to use teh drawing tool.
```
Comment (gevos) [inline src/components/Permalink/Permalink.js]:
```suggestion
      // Redirection to old wkp to use the drawing tool.
```

File: src/components/Share/Share.scss [hunk 4]
```diff
@@ -13,13 +13,21 @@
...
   .ta-mail-icon {
     a {
+      svg {
+        font-size: 23px;
+        margin-bottom: 2px;
+      }
+    }
+  }
+
+  .ta-draw-icon {
```
Comment (gevos) [inline src/components/Share/Share.scss]: The barriere free focus style is not perfect, yet. The box is a little bit to high.

File: src/layers/layerHelper.js (hunk 2 above)
Comment (edellucien) [reply src/layers/layerHelper.js]: In the Permalink I ensure the use of an integer as zoom:
`parseInt(urlParams.z, 10)`
Or would you recommand to parse to Int in the convertToOldZoom func?

Comment (gevos) [reply src/layers/layerHelper.js]: Now, I see it. Yes, I think it is better if you include it in the function `convertToOldZoom()`.

File: src/components/Share/Share.scss (hunk 4 above)
Comment (gevos) [reply src/components/Share/Share.scss]: Fix see https://github.com/geops/trafimage-maps/pull/412

File: src/layers/layerHelper.js (hunk 2 above)
Comment (gevos) [reply src/layers/layerHelper.js]: Short version suggestion with `parseInt()`:
`return zoomEquivalent[Math.min(Math.max(parseInt(zoom, 10), 9), 18)];`

File: src/components/Permalink/Permalink.js [hunk 5]
```diff
@@ -43,13 +45,42 @@ const propTypes = {
...
   initialState: {},
   departuresFilter: undefined,
 };
 
+const redirectToDraw = (drawId, appBaseUrl) => {
+  const urlParams = qs.parse(window.location.search);
+
+  if (urlParams.z) {
+    // Convert the zoom level to match the different scale on the old wkp.
+    urlParams.zoom = layerHelper.convertToOldZoom(parseInt(urlParams.z, 10));
```
Comment (gevos) [inline src/components/Permalink/Permalink.js]:
```suggestion
    urlParams.zoom = layerHelper.convertToOldZoom(urlParams.z);
```

**Pull request merged**